ui/theme/theme.go

```diff
@@ -8,31 +8,61 @@ import (
+	NewContributorIcon      string                 // config.Theme.Icons.NewContributor
+	ContributorIcon         string                 // config.Theme.Icons.Contributor
+	CollaboratorIcon        string                 // config.Theme.Icons.Collaborator
+	MemberIcon              string                 // config.Theme.Icons.Member
+	OwnerIcon               string                 // config.Theme.Icons.Owner
+	UnknownRoleIcon         string                 // config.Theme.Icons.UnknownRole
 }
 
 var DefaultTheme = &Theme{
-	PrimaryBorder:      lipgloss.AdaptiveColor{Light: "013", Dark: "008"},
-	SecondaryBorder:    lipgloss.AdaptiveColor{Light: "008", Dark: "007"},
-	SelectedBackground: lipgloss.AdaptiveColor{Light: "006", Dark: "008"},
-	FaintBorder:        lipgloss.AdaptiveColor{Light: "254", Dark: "000"},
-	PrimaryText:        lipgloss.AdaptiveColor{Light: "000", Dark: "015"},
-	SecondaryText:      lipgloss.AdaptiveColor{Light: "244", Dark: "251"},
-	FaintText:          lipgloss.AdaptiveColor{Light: "007", Dark: "245"},
-	InvertedText:       lipgloss.AdaptiveColor{Light: "015", Dark: "236"},
-	SuccessText:        lipgloss.AdaptiveColor{Light: "002", Dark: "002"},
-	WarningText:        lipgloss.AdaptiveColor{Light: "003", Dark: "003"},
-	ErrorText:          lipgloss.AdaptiveColor{Light: "001", Dark: "001"},
+	PrimaryBorder:           lipgloss.AdaptiveColor{Light: "013", Dark: "008"},
+	SecondaryBorder:         lipgloss.AdaptiveColor{Light: "008", Dark: "007"},
+	SelectedBackground:      lipgloss.AdaptiveColor{Light: "006", Dark: "008"},
+	FaintBorder:             lipgloss.AdaptiveColor{Light: "254", Dark: "000"},
+	PrimaryText:             lipgloss.AdaptiveColor{Light: "000", Dark: "015"},
+	SecondaryText:           lipgloss.AdaptiveColor{Light: "244", Dark: "251"},
+	FaintText:               lipgloss.AdaptiveColor{Light: "007", Dark: "245"},
+	InvertedText:            lipgloss.AdaptiveColor{Light: "015", Dark: "236"},
+	SuccessText:             lipgloss.AdaptiveColor{Light: "002", Dark: "002"},
+	WarningText:             lipgloss.AdaptiveColor{Light: "003", Dark: "003"},
+	ErrorText:               lipgloss.AdaptiveColor{Light: "001", Dark: "001"},
+	NewContributorIconColor: lipgloss.AdaptiveColor{Light: "077", Dark: "077"},
+	ContributorIconColor:    lipgloss.AdaptiveColor{Light: "075", Dark: "075"},
+	CollaboratorIconColor:   lipgloss.AdaptiveColor{Light: "178", Dark: "178"},
+	MemberIconColor:         lipgloss.AdaptiveColor{Light: "178", Dark: "178"},
+	OwnerIconColor:          lipgloss.AdaptiveColor{Light: "178", Dark: "178"},
+	UnknownRoleIconColor:    lipgloss.AdaptiveColor{Light: "178", Dark: "178"},
+	NewContributorIcon:      "󱃱", // U+F10F1 nf-md-hand_heart
+	ContributorIcon:         "󱕾", // U+F157E nf-md-hand_heart_outline
+	CollaboratorIcon:        "", // U+EDA0  nf-fa-handshake_simple
+	MemberIcon:              "", // U+ED9F  nf-fa-handshake_angle
+	OwnerIcon:               "", // U+ED9F  nf-fa-handshake_angle
```

Comment:
I think the `OwnerIcon` should more distinct than the collaborator and member icons. wdyt about "" (\uebb8), "" (\uedcf), "" (\uedd2), "󱇐" (\udb84\uddd0), "󰢏" (\udb84\udf60).